LearningJourney/Modules/Features/JLibrary/JLibraryTests/Sources/Scenes/ObjectivesLisst/Domain/UseCases/ToggleLearnUseCaseTests.swift

```diff
@@ -0,0 +1,49 @@
+//
+//  ToggleLearnUseCaseTests.swift
+//  JLibraryTests
+//
+//  Created by Vinícius Couto on 05/10/21.
+//
+
+import XCTest
+@testable import JLibrary
+
+final class ToggleLearnUseCaseTests: XCTestCase {
+    private let libraryRepositorySpy = LibraryRepositorySpy()
+    private lazy var sut = ToggleLearnUseCase(repository: libraryRepositorySpy)
+
+    func test_execute_itShouldFlipLearnedFlag() {
+        // Given
+        let flagStub = Bool.random()
+        let expectedFlag = !flagStub
+
+        let completionExpectation = expectation(description: "Completion should be called")
+        // When
```

Comment:
Add spacing between your test phases to improve clarity and make a visual difference
```suggestion
        
        // When
        
```